syntax = "proto3";

package Xu;

message TranslateRequest {
    string msg = 1;
};

message TranslateResponse {
    string msg = 1;
};

message AddRequest {
    int32 num1 = 1;
    int32 num2 = 2;
};

message AddResponse{
    int32 result = 1;
};